adagrams/game.py

```diff
@@ -1,13 +1,101 @@
 from random import randint
 
+LETTER_POOL = {
+    "A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2,
+    "G": 3, "H": 2, "I": 9, "J": 1, "K": 1, "L": 4,
+    "M": 2, "N": 6, "O": 8, "P": 2, "Q": 1, "R": 6,
+    "S": 4, "T": 6, "U": 4, "V": 2, "W": 2, "X": 1,
+    "Y": 2, "Z": 1,
+}
+
+SCORE_CHART = {
+    "A": 1, "E": 1, "I": 1, "O": 1, "U": 1, "L": 1, "N": 1, "R": 1, "S": 1, "T": 1,
```

Comment:
To break up long lines of code, we should use multiple lines (see [here](https://peps.python.org/pep-0008/#code-lay-out) in the style guide about using multilines and indentation to create data structures). When I open your code in an editor, I can see that line 2 exceed the maximum line length of 79 characters (as outlined in the style guide [here](https://peps.python.org/pep-0008/#maximum-line-length))
```suggestion
    "A": 1, "E": 1, "I": 1, "O": 1, "U": 1, 
    "L": 1, "N": 1, "R": 1, "S": 1, "T": 1,
```